### DISCUSSION TITLE
Building and Debugging FyroxEngine with Vscode?

### DISCUSSION THREAD

**Discussion by imays76** [FyroxEngine/Fyrox#386] [Q&A]
I installed Rust, VC++ Toolchain and Vscode Extension for Rust. Everything works fine with Vscode for Rust.
I git-cloned FyroxEngine from GitHub, opened it in Vscode, then running `cargo build` works ok.

I tried F5 key(debug-run) but Vscode shows "unable to find executable for ....exe", but I don't know what the build output exe file is.

I tried `cargo run` command on the project root directory, but it shows "error: a bin target must be available for `cargo run`".

What is the ordinary way of building and running(debugging) FyroxEngine with Vscode?

**Comment by mrDIMAS** [comment]
Hi, try `cargo run --package fyroxed --release`. Ideally you should create a project and build it - https://fyrox-book.github.io/fyrox/beginning/scripting.html
